.github/workflows/new-milestone-and-issue-in-other-repo.yml

```diff
@@ -24,6 +25,11 @@ jobs:
     steps:
       - name: Checkout the latest code
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
`fetch-depth` is needed so we can pull the tags as well [as stated in this answer](https://github.com/actions/checkout/issues/701#issuecomment-1133937950).

This is why [this error popped up in the original issue](https://github.com/checkstyle/checkstyle/actions/runs/3813358660/jobs/6487062297#step:3:8):
```
error: pathspec 'checkstyle-10.6.0' did not match any file(s) known to git
```